### PR TITLE
[CARBONDATA-2448] Adding compacted segments to load and alter events

### PR DESCRIPTION
**Pull request by dhatchayani**
Adding compacted segments to load and alter table events
 - [ ] Any interfaces changed?
 
 - [ ] Any backward compatibility impacted?
 
 - [ ] Document update required?

 - [x] Testing done
        Manual Testing
       
 - [ ] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA.

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Failed with Spark 2.2.1, Please check CI http://88.99.58.216:8080/job/ApacheCarbonPRBuilder/4526/

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/4758/

File: integration/spark-common/src/main/scala/org/apache/carbondata/events/AlterTableEvents.scala [hunk 1]
```diff
@@ -166,7 +166,8 @@ case class AlterTableCompactionPreEvent(sparkSession: SparkSession,
 case class AlterTableCompactionPostEvent(sparkSession: SparkSession,
     carbonTable: CarbonTable,
     carbonMergerMapping: CarbonMergerMapping,
-    mergedLoadName: String) extends Event with AlterTableCompactionEventInfo
+    mergedLoadName: String,
+    compactedLoads: java.util.List[String]) extends Event with AlterTableCompactionEventInfo
```
Comment (manishgupta88) [inline integration/spark-common/src/main/scala/org/apache/carbondata/events/AlterTableEvents.scala]: if compactedLoads list is added then you can remove mergedLoadName

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/4761/

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/5694/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/5696/

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/4763/

**Comment by ravipesala** [comment]
SDV Build Fail , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/4772/

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/4774/

**Comment by manishgupta88** [comment]
LGTM

**Pull request closed without merging**